docs/tutorials/NEB/neb_tutorial.md

```diff
@@ -287,6 +289,6 @@ In many cases, particularly when lighter atoms are involved, the energy barrier
  
 The simplest way to do this is in the harmonic approximation. You need to estimate the harmonic zero-point energy at the start, transition and final configurations.
 
-For the particular case (ammonia) we have been looking at, it turns out that to actually explain the observed transition rate between the two 'umbrella' configurations, we need to go beyond the harmonic approximation ([see Wikipedia](https://en.wikipedia.org/wiki/Pyramidal_inversion#Quantum_effects)).
+For the particular case (ammonia) we have been looking at, it turns out that to actually explain the observed transition rate between the two 'umbrella' configurations, we need to go beyond the harmonic approximation ([see Wikipedia](https://en.wikipedia.org/wiki/Pyramidal_inversion#Quantum_effects)).  *[This is a bit obscure. Is it important for tutorial?]*
```

Comment:
I wanted something at the end to tell the user what the next steps might be and to caution against taking the NEB barrier as gospel. Often the NEB barrier will need zero point energy corrections. These are most often done in the harmonic approximation. But even that wouldn't be enough to get an accurate barrier estimate for this NH3 example. 

Perhaps the ZPE correction instructions should be in a separate tutorial, but I wanted to flag it here as it's relevant for NEB barrier estimates.

Reply:
I get the point, but I think it's getting away from the concept of a tutorial on how to run NEB calculations towards a tutorial on calculating the energy barrier in NH3. I would simplify to a note that highlights that the NEB will provide a Delta E (this is exact), while the transition rates will depend on Delta E_0. The ZPE corrections may be significant  [presumably if the motion is dominated by H movement?]

Reply:
```suggestion
TODO: replace the section on zero-point energy corrections by a note that highlights that the NEB will provide a Delta E (this is exact), while the transition rates will depend on Delta E_0.
```